### PR TITLE
Host Details Locator Cleanup

### PR DESCRIPTION
**Pull request by sambible**
Replace individual locators with HostDetailsCard widget, and remove unnecessary and unused entity methods.

These cards can easily be accessed through the get_details method instead. 

Probably also worth stating that these cards aren't currently under test, and this PR doesn't remedy that, it just cleans up the locators and entities that were written when the new Host Details UI was made originally.

### CODE REVIEW

**Review by omkarkhatavkar — APPROVED** [review]
ACK, nice work @sambible, pending some test result, please run PRT

File: airgun/entities/host_new.py [hunk 1]
```diff
@@ -399,43 +399,6 @@ def get_tracer(self, entity_name):
...
-        self.browser.plugin.ensure_page_safe()
-        return view.details.hw_properties.read()
-
-    def get_provisioning_templates(self, entity_name):
-        view = self.navigate_to(self, 'NewDetails', entity_name=entity_name)
-        view.wait_displayed()
-        self.browser.plugin.ensure_page_safe()
-        d = view.details.read()
-        return d['provisioning_templates']['templates_table']
-
```
Comment (omkarkhatavkar) [inline airgun/entities/host_new.py:438]: all these methods are been used as part of `HostDetailsCard`?

Comment (jyejare) [reply airgun/entities/host_new.py:438]: I think we need to also open a parallel PR in robottelo to remove these function usages as well! Do we have any, if yes please link here

**Comment by jyejare** [comment]
@sambible Hello, Could you please run PRT as @omkarkhatavkar has requested above! Its helpful in merging / reviewing your PR !

File: airgun/entities/host_new.py (hunk 1 above)
Comment (omkarkhatavkar) [reply airgun/entities/host_new.py:438]: @LadislavVasina1 what's your view here? are these functions are necessary?

Comment (LadislavVasina1) [reply airgun/entities/host_new.py:438]: @omkarkhatavkar I wrote those entities to be used by the user as an oneliner that could help him/her simplify the development of UI tests in a way that they don't have to go through the structure of dictionary returned by `view.details.read()` and look for specific keys that they would need to achieve desired output. In my opinion they could stay there as they can probably be useful to someone in future.

**Comment by sambible** [comment]
Alright, sorry this took so long, but here it is: This is a printout of what fetching 'details' with the get_details method returns, which includes most of the cards that I've updated here
`{'details': {'card_collapse_switch': 'Collapse all cards', 'system_properties': {'title': 'System properties', 'sys_properties': {'name': 'xmwdesrplz.6w6ge9ioee', 'domain': '6w6ge9ioee', 'model': 'Not available', 'tracer': 'Not installed', 'host_group': 'Not available', 'host_owner': 'host_qXY8N20cXw', 'owner_type': 'User', 'organization': 'WaHEeGn', 'location': 'gVpziz'}}, 'operating_system': {'title': 'Operating system', 'details': {'architecture': 'sdUnrsByvcZ', 'os': 'bHdkefjf 2', 'boot_time': 'Not available', 'kernel_release': 'Not available'}}, 'provisioning': {'title': 'Provisioning', 'details': {'build_duration': 'Not available', 'token': '9d5272a9-7084-4924-8bd7-2ba730eed6c6', 'pxe_loader': 'Not available'}}, 'bios': {'title': 'BIOS', 'details': {'vendor': 'Not available', 'version': 'Not available', 'release_date': 'Not available'}}, 'registration_details': {'details': {}}, 'hw_properties': {'details': {}}, 'provisioning_templates': {'title': 'Provisioning templates', 'templates_table': [{'column0': '', 'column1': 'Linux host_init_config default', 'column2': ''}]}, 'installed_products': {}, 'networking_interfaces': {'title': 'Networking interfaces', 'networking_interfaces_accordion': {}, 'edit_interfaces': 'Edit interfaces'}, 'networking_interface': {'title': 'System properties'}, 'virtualization': {'details': {}}}}`

This is pretty barebones, since the host didn't have too much info, but you can see that it successfully reads all the cards like it would have before. Hopefully this is enough confidence to merge this PR :)

File: airgun/entities/host_new.py (hunk 1 above)
Comment (omkarkhatavkar) [reply airgun/entities/host_new.py:438]: @sambible based on  @LadislavVasina1 comments, can you please keep these functions? and revert it.

**Comment by omkarkhatavkar** [comment]
> Alright, sorry this took so long, but here it is: This is a printout of what fetching 'details' with the get_details method returns, which includes most of the cards that I've updated here `{'details': {'card_collapse_switch': 'Collapse all cards', 'system_properties': {'title': 'System properties', 'sys_properties': {'name': 'xmwdesrplz.6w6ge9ioee', 'domain': '6w6ge9ioee', 'model': 'Not available', 'tracer': 'Not installed', 'host_group': 'Not available', 'host_owner': 'host_qXY8N20cXw', 'owner_type': 'User', 'organization': 'WaHEeGn', 'location': 'gVpziz'}}, 'operating_system': {'title': 'Operating system', 'details': {'architecture': 'sdUnrsByvcZ', 'os': 'bHdkefjf 2', 'boot_time': 'Not available', 'kernel_release': 'Not available'}}, 'provisioning': {'title': 'Provisioning', 'details': {'build_duration': 'Not available', 'token': '9d5272a9-7084-4924-8bd7-2ba730eed6c6', 'pxe_loader': 'Not available'}}, 'bios': {'title': 'BIOS', 'details': {'vendor': 'Not available', 'version': 'Not available', 'release_date': 'Not available'}}, 'registration_details': {'details': {}}, 'hw_properties': {'details': {}}, 'provisioning_templates': {'title': 'Provisioning templates', 'templates_table': [{'column0': '', 'column1': 'Linux host_init_config default', 'column2': ''}]}, 'installed_products': {}, 'networking_interfaces': {'title': 'Networking interfaces', 'networking_interfaces_accordion': {}, 'edit_interfaces': 'Edit interfaces'}, 'networking_interface': {'title': 'System properties'}, 'virtualization': {'details': {}}}}`
> 
> This is pretty barebones, since the host didn't have too much info, but you can see that it successfully reads all the cards like it would have before. Hopefully this is enough confidence to merge this PR :)

Thanks @sambible !

**Pull request merged**